bolt-design-system/bolt
# Update Status Board To Support Optional "Pattern Alias" Name

## Jira
N/A

## Summary
- Updates the component status board logic to support using a new (optional) `pattern-alias` config when automatically matching up a component with the component's main viewall page in Pattern Lab.

## Details
- Also reverts adding number prefixes to several component folders in PL since this update eliminates the need to manually order these folders.

## How to test
- Confirm the status board builds as expected
- Confirm the status board links for button, button group, background, and background shape all link to the correct viewall page in Pattern Lab correctly
- Confirm the nav text for `button group` and `background shapes` has reverted back to showing `backgrounds-shapes` and `buttons-group` -- a workaround till the underlying PL bug for this is fixed.
